api/app/src/external/commonwell/link/shared.ts

```diff
@@ -0,0 +1,48 @@
+import { Person } from "@metriport/commonwell-sdk";
+import { apiUrl } from "../api";
+import {
+  Patient,
+  PatientData,
+  PatientExternalData,
+  PatientExternalDataEntry,
+} from "../../../models/medical/patient";
+import { PatientDataCommonwell } from "../patient-shared";
+import { MedicalDataSource } from "../..";
+
+export const createPatientLink = (personId: string, patientId: string): string => {
+  return `${apiUrl}/v1/person/${personId}/patientLink/${patientId}/`;
+};
+
+export const commonwellPersonLinks = (persons: Person[]): Person[] => {
```

Comment:
Could use `.flatMap(filterTruthy)` instead, it return an array without null/undefined items, and its strongly typed.